### PR TITLE
PXC-4323 Add telemetry topics (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
new file:   docs/telemetry.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+## Disable telemetry
+
+Starting with Percona XtraDB Cluster 8.0.34, the telemetry is enabled by default. If you decide to not send usage data to Percona, you can set the `PERCONA_TELEMETRY_DISABLE=1` environment variable for either the root user or in the operating system prior to the installation process.
+
+=== "Debian-derived distribution"
+
+    Add the environment variable before the install process.
+
+    ```{.bash data-prompt="$"}
```
Comment (eleo007) [inline docs/telemetry.md:33]: Command is for percona-server, please change it to percona-xtradb-cluster

File: docs/telemetry.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    $ sudo PERCONA_TELEMETRY_DISABLE=1 apt install percona-server-server
+    ```
+
+=== "Red Hat-derived distribution"
+
+    Add the environment variable before the install process.
+    
+    ```{.bash data-prompt="$"}
+    $ sudo PERCONA_TELEMETRY_DISABLE=1 yum install percona-server-server
```
Comment (eleo007) [inline docs/telemetry.md]: The same here, package is  `percona-xtradb-cluster`

File: docs/telemetry.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    ```
+
+=== "DOCKER"
+
+    Add the environment variable when running a command in a new container.
+    
+    ```{.bash data-prompt="$"}
+    $ docker run percona/percona-server -e PERCONA_TELEMETRY_DISABLE=1
```
Comment (eleo007) [inline docs/telemetry.md]: I would use command that's more related to cluster:
docker run -d \
  -e MYSQL_ROOT_PASSWORD=test1234# \
 -e PERCONA_TELEMETRY_DISABLE=1 \
  -e CLUSTER_NAME=pxc-cluster1 \
  --name=pxc-node1 \
  percona/percona-xtradb-cluster:8.0

File: docs/telemetry.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Telemetry on Percona XtraDB Cluster
+
+Percona telemetry fills in the gaps in our understanding of how you use Percona XtraDB Cluster to improve our products. Participation in the anonymous program is optional. You can opt-out if you prefer not to share this information.
+
+## What information is collected
+
+At this time, Percona XtraDB Cluster collects only information about the installation environment. Future releases may add additional metrics.
+
+Be assured that the access to this raw data is rigorously controlled. Percona does not collect personal data. All data is anonymous and cannot be traced to a specific user. For more information read our [Percona Privacy statement] to learn more on our privacy practices.
```
Comment (alina-derkach-oaza) [inline docs/telemetry.md]:
```suggestion
Be assured that access to this raw data is rigorously controlled. Percona does not collect personal data. All data is anonymous and cannot be traced to a specific user. To learn more about our privacy practices, read our [Percona Privacy statement].
```

File: docs/telemetry.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+## Disable telemetry
+
+Starting with Percona XtraDB Cluster 8.0.34, telemetry is enabled by default. If you decide to not send usage data to Percona, you can set the `PERCONA_TELEMETRY_DISABLE=1` environment variable for either the root user or in the operating system prior to the installation process.
```
Comment (alina-derkach-oaza) [inline docs/telemetry.md]:
```suggestion
Starting with Percona XtraDB Cluster 8.0.34, telemetry is enabled by default. If you decide not to send usage data to Percona, you can set the `PERCONA_TELEMETRY_DISABLE=1` environment variable for either the root user or in the operating system prior to the installation process.
```

**Review by kamil-holubicki — APPROVED** [review]

**Review by alina-derkach-oaza — APPROVED** [review]

File: docs/telemetry.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+
+At this time, telemetry is added only to the Percona packages and Docker images. Percona XtraDB Cluster collects only information about the installation environment. Future releases may add additional metrics.
+
+Be assured that access to this raw data is rigorously controlled. Percona does not collect personal data. All data is anonymous and cannot be traced to a specific user. To learn more about our privacy practices, read our [Percona Privacy statement].
+
+An example of the data collected is the following:
+
+```JSON
+[{"id" : "c416c3ee-48cd-471c-9733-37c2886f8231",
+"product_family" : "PRODUCT_FAMILY_PS",
```
Comment (eleo007) [inline docs/telemetry.md]: PRODUCT_FAMILY_PXC

File: docs/install-index.md [hunk 7]
```diff
@@ -3,6 +3,8 @@
 Install Percona XtraDB Cluster on all hosts that you are planning to use as cluster nodes
 and ensure that you have root access to the MySQL server on each one.
 
+We gather [Telemetry data] in the Percona packages and Docker images.
```
Comment (eleo007) [inline docs/install-index.md:6]: Hi,
Could you please also add it to the pages:
* [Debian or Ubuntu](apt.md#apt) 
* Red Hat or CentOS](yum.md#yum)
* Running Percona XtraDB Cluster in a Docker Container

File: docs/telemetry.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+An example of the data collected is the following:
+
+```JSON
+[{"id" : "c416c3ee-48cd-471c-9733-37c2886f8231",
+"product_family" : "PRODUCT_FAMILY_PXC",
+"instanceId" : "6aef422e-56a7-4530-af9d-94cc02198343",
+"createTime" : "2023-10-16T10:46:23Z",
+"metrics":
+[{"key" : "deployment","value" : "PACKAGE"},
+{"key" : "pillar_version","value" : "8.0.33-25-"},
```
Comment (eleo007) [inline docs/telemetry.md]: Just noticed: 8.0.34-26-1

**Review by eleo007 — APPROVED** [review]

**Pull request merged**